File: ts/wam/4218.ts
export const WamLockFolderUnlock = 4218 as const

export interface LockFolderUnlockEvent {
  /** field 1, wire `landing_surface` */
  landingSurface?: LANDINGSURFACE
  /** field 2, wire `total_chat_count` */
  totalChatCount?: number
  /** field 3, wire `unlock_entry_point` */
  unlockEntryPoint?: UNLOCKENTRYPOINT
}

File: rust/wam/4218.rs
pub const WAM_LOCK_FOLDER_UNLOCK: u32 = 4218;

#[derive(Debug, Default)]
pub struct LockFolderUnlockEvent {
    /// field 1, wire `landing_surface`
    pub landing_surface: Option<LANDINGSURFACE>,
    /// field 2, wire `total_chat_count`
    pub total_chat_count: Option<i64>,
    /// field 3, wire `unlock_entry_point`
    pub unlock_entry_point: Option<UNLOCKENTRYPOINT>,
}